pkg/controller/backup_controller.go

```diff
@@ -263,8 +292,19 @@ func (c *backupController) processBackup(key string) error {
 		return nil
 	}
 
-	c.backupTracker.Add(request.Namespace, request.Name)
-	defer c.backupTracker.Delete(request.Namespace, request.Name)
+	// Set NextCheckTime to dontProcessTime so that checkAndFinalizeBackupsLoop does not process it after being added to
+	// the tracker
+	request.NextCheckTime = dontProcessTime
```

Comment:
A little confused here. Since backupTracker is mainly used to track uploading progress and uploading partial failure backups. Looks like no need to add the backup in progress and set an invalid check time.